overturemaps/cli.py

```diff
@@ -27,17 +27,26 @@ def get_writer(output_format, path, schema):
     elif output_format == "geojsonseq":
         writer = GeoJSONSeqWriter(path)
     elif output_format == "geoparquet":
-        # Update the geoparquet metadata to remove the file-level bbox which
-        # will no longer apply to this file. Since we cannot write the field at
-        # the end, just remove it as it's optional. Let the per-row bounding
-        # boxes do all the work.
+        # don't remove per row bbox and add covering spec for spatial filters
         metadata = schema.metadata
         geo = json.loads(metadata[b"geo"])
-        for column in geo["columns"].values():
-            column.pop("bbox")
```

Comment:
These need to stay since the file-level bounding box might no longer be correct. It can be computed and added to the metadata before the file is closed, but couldn't be done here.